web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -268,8 +278,17 @@ def mentor_program_families(self):
             self.program_family, flat=True).distinct()
 
     def location_choices(self):
-        return self.user_query.values(
+        locations = self.user_query.values(
             **_location_lookups(self.location_path)).distinct()
+        remote_location_fields = dict([("location_" + field, F(field))
+                                       for field in LOCATION_FIELDS])
+        remote_location = Location.objects.filter(
+            name="Remote").values(**remote_location_fields).first()
+        locations_list = list(locations)
+        has_remote_location = _check_remote_location(locations_list)
+        if remote_location and not has_remote_location: locations_list.append(
```

Comment:
Not a blocker, but it would be a little better to check `has_remote_location` before going to the DB to fetch the remote location instance.

Reply:
+1